paddle/fluid/operators/tensorrt/tensorrt_engine_op_test.cc

```diff
@@ -99,7 +99,7 @@ TEST(TensorRTEngineOp, manual) {
   SetAttr<std::string>(engine_op_desc.Proto(), "subgraph",
                        block_->SerializeAsString());
   SetAttr<int>(engine_op_desc.Proto(), "max_batch_size", 2);
-  SetAttr<int>(engine_op_desc.Proto(), "workspace_size", 2 << 10);
+  SetAttr<int>(engine_op_desc.Proto(), "workspace_size", 2 << 20);
```

Comment:
Why change to `2<<20` ?

Reply:
We should change it to 1 << 20 (the Official default). 1 << 10 is too small here.